azkaban-web-server/src/web/js/azkaban/view/exflow.js

```diff
@@ -447,6 +449,10 @@ var updaterFunction = function() {
 			// 2 min updates
 			setTimeout(function() {updaterFunction();}, 2*60*1000);
 		}
+		else if (data.status == "KILLING") {
+			// 20 s updates
+			setTimeout(function() {updaterFunction();}, 30*1000);
```

Comment:
so this is interval between each updaterFunction? can you help me understand better why it's 30 secs? also comment says 20s